canonical-web-and-design/greenhouse-exporter
# [Front end] Add testing dependencies

## Done
Added
- Jest
- Testing Library
- [msw](https://mswjs.io/) (for mocking api calls)
## QA
- Run `yarn test-js` and see the test passes
- `dotrun --env DEBUG_USER_EMAIL=peter.mahnke@canonical.com`
- Verify everything is still running in `http://localhost:8501/`
